diff --git a/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js b/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
index 87cb380..ff9306d 100644
--- a/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
+++ b/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
@@ -45,22 +45,6 @@ function warnAboutTSVersion(parseSettings, passedLoggerFn) {
     if (isRunningSupportedTypeScriptVersion || warnedAboutTSVersion) {
         return;
     }
-    if (passedLoggerFn ||
-        // See https://github.com/typescript-eslint/typescript-eslint/issues/7896
-        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-        (typeof process === 'undefined' ? false : process.stdout?.isTTY)) {
-        const border = '=============';
-        const versionWarning = [
-            border,
-            'WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.',
-            'You may find that it works just fine, or you may not.',
-            `SUPPORTED TYPESCRIPT VERSIONS: ${SUPPORTED_TYPESCRIPT_VERSIONS}`,
-            `YOUR TYPESCRIPT VERSION: ${ACTIVE_TYPESCRIPT_VERSION}`,
-            'Please only submit bug reports when using the officially supported version.',
-            border,
-        ].join('\n\n');
-        parseSettings.log(versionWarning);
-    }
     warnedAboutTSVersion = true;
 }
 //# sourceMappingURL=warnAboutTSVersion.js.map
\ No newline at end of file
